client/src/main/java/org/jboss/fuse/mvnd/client/DefaultClient.java

```diff
@@ -73,14 +81,36 @@ public static void main(String[] argv) throws Exception {
         }
     }
 
+    private static void install(boolean overwrite) {
+        final Properties buildProps = loadBuildProperties();
+        final String version = buildProps.getProperty("version");
+        final String rawZipUri = System.getProperty("mvnd.zip.uri", "https://github.com/ppalaga/mvnd/releases/download/"+ version +"/mvnd-" + version + ".zip");
```

Comment:
Shouldn't this be modified to point to the `mvndaemon/mvnd` repo instead ?

Reply:
I changed it in one of the subsequent commits https://github.com/mvndaemon/mvnd/pull/30/files#diff-86f08db3461cd372b91b0c70cd8606c5R85